src/components/AcceptedValuesField/AcceptedValuesField.js

```diff
@@ -39,20 +41,26 @@ export const AcceptedValuesField = ({
     data.forEach(item => {
       acceptedValues = {
         ...acceptedValues,
-        [item.id]: item.name,
+        [item.id]: optionTemplate ? updateValueWithTemplate(item, optionTemplate) : item[optionValue],
       };
     });
 
     return acceptedValues;
   };
 
+  const updateListOption = data => data.map(option => ({
```

Comment:
Just suggestion
```suggestion
  const getListOptions= data => data.map(option => ({
```

Reply:
As for me we are updating options, that’s why I named it this way